pg-ragtime/src/pg/ragtime.clj

```diff
@@ -0,0 +1,29 @@
+(ns pg.ragtime
```

Comment:
The draft looks solid, thank you. Just a note: you can borrow most of the code from here: https://github.com/weavejester/ragtime/blob/master/next-jdbc/src/ragtime/next_jdbc.clj

Reply:
> Just a note: you can borrow most of the code from here: https://github.com/weavejester/ragtime/blob/master/next-jdbc/src/ragtime/next_jdbc.clj

That's a very good point I didn't think of. `ragtime.next-jdbc` is an excellent starting point for `pg.ragtime`.